bubble_sort.rb

```diff
@@ -0,0 +1,11 @@
+def bubble_sort(input_arr)
+  input_arr.length.times do
+    input_arr.each_with_index do |num, index|
+      next unless input_arr[index + 1]
+
+      input_arr[index], input_arr[index + 1] = input_arr[index + 1], input_arr[index] if num > input_arr[index + 1]
+    end
+  end
+  input_arr
+end
+puts bubble_sort([4, 3, 78, 2, 0, 2])
```

Comment:
Line 11. where you have 'puts bubble_sort([4, 3, 78, 2, 0, 2])' it is a debugging code. Kindly deleted or move to another branch. :wrench: 

![debugging_requirement](https://user-images.githubusercontent.com/64173189/107528176-8c16e600-6b87-11eb-9558-26441a83a87b.png)